languages/elixir/exercises/concept/enum/.docs/after.md

```diff
@@ -0,0 +1,132 @@
+`Enum` is a very useful module that provides a set of algorithms for working with enumerables. It offers:
+
+- sorting ([`sort/2`][enum-sort/2], [`sort_by/2`][enum-sort_by/2], [`reverse/2`][enum-reverse/2], [`shuffle/1`][enum-shuffle/1]),
+- filtering ([`filter/2`][enum-filter/2], [`reject/2`][enum-reject/2], [`uniq/1`][enum-uniq/1], [`uniq_by/2`][enum-uniq_by/2]),
+- grouping ([`chunk_by/2`][enum-chunk_by/2], [`chunk_every/2`][enum-chunk_every/2], [`chunk_while/4`][enum-chunk_while/4], [`group_by/3`][enum-group_by/3]),
+- counting ([`count/2`][enum-count/2], [`frequencies/1`][enum-frequencies/1], [`frequencies_by/2`][enum-frequencies_by/2])
+- searching ([`find/3`][enum-find/3], [`find_index/2`][enum-find_index/2], [`find_value/3`][enum-find_value/3]),
+- finding min/max values ([`min/3`][enum-min/3], [`max/3`][enum-max/3], [`min_by/4`][enum-min_by/4], [`max_by/4`][enum-max_by/4], [`min_max/2`][enum-min_max/2], [`min_max_by/3`][enum-min_max_by/3]),
+- reducing ([`reduce/3`][enum-reduce/3], [`reduce_while/3`][enum-reduce_while/3], [`sum/1`][enum-sum/1]),
+- and much more.
```

Comment:
I found this list a bit overwhelming. Maybe list the most useful ones and then link to a document that lists all methods?

Reply:
My goal was to dazzle people with the richness of the standard library :) but if it's overwhelming, then I'll keep it to just the essentials.

Reply:
Well, dazzling it did :) To me it felt a bit overwhelming. You could consider splitting it up then. Maybe just list the important ones first, and the minor ones later?

Reply:
I limited the list to just the most important ones per operation.